src/windows/wslaservice/exe/WSLAVirtualMachine.cpp

```diff
@@ -162,12 +173,8 @@ void WSLAVirtualMachine::Start()
     // Configure backing page size, fault cluster shift size, and cold discard hint size to favor density (lower vmmem usage).
     //
     // N.B. Cold discard hint size should be a multiple of the fault cluster shift size.
-    //
-    // N.B. This is only done on builds that have the fix for the VID deadlock on partition teardown.
-    if ((m_windowsVersion.BuildNumber >= WindowsBuildNumbers::Germanium) ||
-        (m_windowsVersion.BuildNumber >= WindowsBuildNumbers::Cobalt && m_windowsVersion.UpdateBuildRevision >= 2360) ||
-        (m_windowsVersion.BuildNumber >= WindowsBuildNumbers::Iron && m_windowsVersion.UpdateBuildRevision >= 1970) ||
-        (m_windowsVersion.BuildNumber >= WindowsBuildNumbers::Vibranium_22H2 && m_windowsVersion.UpdateBuildRevision >= 3393))
+    const auto windowsVersion = wsl::windows::common::helpers::GetWindowsVersion();
+    if (windowsVersion.BuildNumber >= WindowsBuildNumbers::Germanium)
```

Comment:
nit: I'd prefer to keep th previous checks to avoid hitting the deadlock if we ever run on older builds

Reply:
The relaxing of this check means we will fall back to the “safe but less memory efficient” behavior more often. It doesn’t prevent the deadlock.